test/test_alias.cpp

```diff
@@ -172,7 +172,7 @@ TEST_F(AliasAnalysisTest, View_SplitExpandedBroadcast) {
 
   optimization::AliasAnalysisResult alias_analysis =
       optimization::findAliases(&fusion);
-  EXPECT_EQ(alias_analysis.findRoot(out), expand_out);
+  EXPECT_EQ(alias_analysis.findRoot(out), out);
```

Comment:
Why is `findRoot(out)` not equal to `in` in this case? Is this just because of #1126? If so, maybe we should add a comment that ideally we would alias the input here, and refer to that issue.

Reply:
> Is this just because of #1126?

No, no-alias is the right answer here. `in`'s shape is `{4,5},stride={5,1}` and `out`'s shape is `{40,3}`. No matter what strides we pick for `out`, there has to be a memcpy. 

Unlike this case, `View_ForwardExpandedBroadcast` can be made an alias as long as `out`'s shape is `{20,6},strides={1,0}`.

Reply:
Ah ok, got it. Of course. Thanks for the explanation.